src/FormSection/index.js

```diff
@@ -9,15 +9,17 @@ const FormSection = ({ title, children }) => (
   <div className="nds-formSection fontFamily--body">
     <Row alignItems="center" gapSize="xs">
       <Row.Item shrink>
-        <h3 className="fontColor--secondary fontSize--xs fontFamily--body">{title}</h3>
+        <h3 className="fontColor--secondary fontSize--xs fontFamily--body">
+          {title}
+        </h3>
       </Row.Item>
       <Row.Item>
-        <div style={{ width: "100%", height: "var(--space-m)" }}>
+        <div style={{ width: "100%", height: "var(--font-size-default)" }}>
```

Comment:
`--font-size` vars scale; `--space` vars do not.

This is intentional, as text zoom should not increase gutters